src/UI/Window.h

```diff
@@ -60,7 +60,7 @@ class cWindow
 		wtDropSpenser = 3,  // Dropper or Dispenser
 		wtEnchantment = 4,
 		wtBrewery     = 5,
-		wtNPCTrade    = 6,
+		wtVillagerTrade = 6,
```

Comment:
This symbol was already exported to Lua. Are you sure you want to rename it just for the sake of renaming?

Reply:
Yes <3